gcloud/datastore/client.py

```diff
@@ -127,8 +128,42 @@ def __init__(self, dataset_id=None, namespace=None, connection=None):
         if connection is None:
             connection = get_connection()
         self.connection = connection
+        self._batch_stack = _LocalStack()
```

Comment:
Is it expensive to create an instance? Is it worth holding off (via a lazy-loaded property) until it is used?